app_builder/lib/app_builder.ex

```diff
@@ -1,5 +1,213 @@
+    File.mkdir_p!(erts_destination_source)
+
+    erts_source
+    |> Path.join("bin")
+    |> File.cp_r!(erts_destination_source, fn _, _ -> false end)
+
+    _ = File.rm(Path.join(erts_destination_source, "erl"))
+    _ = File.rm(Path.join(erts_destination_source, "erl.ini"))
+
+    erts_destination_source
+    |> Path.join("erl")
+    |> File.write!(~S"""
+    #!/bin/sh
+    SELF=$(readlink "$0" || true)
+    if [ -z "$SELF" ]; then SELF="$0"; fi
+    BINDIR="$(cd "$(dirname "$SELF")" && pwd -P)"
+    ROOTDIR="${ERL_ROOTDIR:-"$(dirname "$(dirname "$BINDIR")")"}"
+    EMU=beam
+    PROGNAME=$(echo "$0" | sed 's/.*\///')
+    export EMU
+    export ROOTDIR
+    export BINDIR
+    export PROGNAME
+    exec "$BINDIR/erlexec" ${1+"$@"}
+    """)
+    executable!(Path.join(erts_destination_source, "erl"))
+
+    # Copy lib
+    erts_destination_lib = Path.join(release.path, "lib")
+    File.mkdir_p!(erts_destination_lib)
+
+    erts_lib_dir
+    |> File.cp_r!(erts_destination_lib, fn _, _ -> false end)
+
+    # copy start.boot to <resource_path>/rel/bin
+    erts_destination_bin =  Path.join(release.path, "/bin")
+    start_boot_file = Path.join(erts_destination_bin, "start.boot")
+    File.mkdir_p!(erts_destination_bin)
+
+    erts_bin_dir
+    |> Path.join("start.boot")
+    |> File.cp!(start_boot_file, fn _, _ -> false end)
+
+    %{release | erts_source: erts_source}
+  end
+
+  @erts_bin [~s[ERTS_BIN="$ERTS_BIN"], ~s[ERTS_BIN=!ERTS_BIN!]]
+
+  defp replace_erts_bin(contents, release, new_path) do
+    if release.erts_source do
+      String.replace(contents, @erts_bin, ~s[ERTS_BIN=#{new_path}])
+    else
+      contents
+    end
+  end
+
+  @doc """
+  Copies elixir into the release.
+  """
+  @spec copy_elixir(Mix.Release.t()) :: Mix.Release.t()
+  def copy_elixir(release) do
+    include_executables_for = Keyword.get(release.options, :include_executables_for, [:unix])
+
+    elixir_bin_path = Application.app_dir(:elixir, "../../bin")
+    bin_path = Path.join(release.path, "bin")
+    File.mkdir_p!(bin_path)
+
+    # download and unzip
+    standalone_destination = Path.join(release.path, "elixir")
+    download_elixir_at_destination(standalone_destination)
+
+    # patch elixir file to look for the right erts <resource_path>/rel/releases/#{release.version}/elixir
+    patch_elixir(include_executables_for, release,
+      fn filename ->
+        Path.join(elixir_bin_path, filename)
+      end,
+      fn filename ->
+        Path.join(release.version_path, filename)
+      end
+    )
+
+    # patch elixir file to look for the right erts <resource_path>/rel/elixir/bin/elixir
+    patch_elixir(include_executables_for, release,
+      fn filename ->
+        Path.join([standalone_destination, "bin", filename])
+      end,
+      fn filename ->
+        Path.join([standalone_destination, "bin", filename])
+      end
+    )
+
+    release
+  end
+
+  @elixir_standalone_version "1.13.2"
```

Comment:
Let's make this an argument to `copy_elixir`, so we can pass the version from the mix.exs and we don't have to manage it across multiple files. :)

Reply:
I changed it as you suggest; it will be `Standalone.copy_elixir("1.13.2")`; Is there a way to get the exact version? To use it instead of hardcoded.